src/main/flight/pid.c

```diff
@@ -774,6 +775,20 @@ float pidGetAirmodeThrottleOffset(void)
 }
 #endif
 
+static FAST_CODE_NOINLINE void disarmOnImpact(const float delta, const float errorRate)
+{
+    // if all sticks are within 5% of center, and throttle low, check acc and gyro for impacts
+    // threshold should be high enough to avoid unwanted disarms in the air on throttle chops
+    if (isAirmodeActivated() && getMaxRcDeflectionAbs() < 0.05f && mixerGetRcThrottle() < 0.05f) {
+        if (acc.accMagnitude > pidRuntime.ezLandingDisarmThreshold || 
+            (fabsf(delta) > (pidRuntime.crashDtermThreshold) && fabsf(errorRate) > pidRuntime.crashGyroThreshold)) {
```

Comment:
These two brackets do nothing
```suggestion
            (fabsf(delta) > pidRuntime.crashDtermThreshold && fabsf(errorRate) > pidRuntime.crashGyroThreshold)) {
```